inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp

```diff
@@ -211,7 +187,7 @@ void MKLDNNPlugin::MKLDNNInferRequest::GetBlob(const char *name, InferenceEngine
         // in situation if we push output blobs as inputs for next network (in Hetero plugin)
         // it may be that output tensor desc will be different from real input tensor desc for next network
         // because the optimal descriptor was chosen (e.g. inPlace case for Split node)
-        auto currBlockDesc = InferenceEngine::BlockingDesc(desc.getDims(), desc.getBlockingDesc().getOrder());
+        auto currBlockDesc = InferenceEngine::BlockingDesc(desc.getBlockingDesc().getBlockDims(), desc.getBlockingDesc().getOrder());
```

Comment:
It looks like not refactoring, but changing the logic. Could you please clarify why the change is legal?

Reply:
At this moment previous version and current version are similar.
When we create ```output node``` we don't consider real tensor descriptor and create "default" ```MKLDNNMemoryDesc``` based only on layout.
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/mkldnn_input_node.cpp#L73
And blocked dimensions are equals to tensor dimensions.
According to ```BlockingDesc``` constructor its arguments are ```block_dims``` and ```order```.
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/inference_engine/ie_layouts.cpp#L238
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp#L1021